force-app/main/default/translations/ca.translation-meta.xml

```diff
@@ -997,39 +997,39 @@
         <name>stgHCReciprocalRelFemaleInactiveFix</name>
```

Comment:
Assuming we need to reset the translation for this label too?

Reply:
Done